README.md

```diff
@@ -69,6 +69,13 @@
 
 To get a local copy up and running, follow these steps.
 
+To get this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ambrose-kibet/Budget-App.git
+```
+
```

Comment:
- [ ] I am making this comment here because this is the only file you have changed in your PR, always make sure you create a PR with the branch with your updated files, currently, your design does not look exactly like the given design, as mentioned in the project requirement, kindly keep to colour and spacing
your design :point_down: 
![Screenshot from 2023-06-15 22-10-22](https://github.com/ambrose-kibet/Budget-App/assets/66097371/55033590-3b86-4f2d-9d47-7437c4c8813b)
given design :point_down: 
![Screenshot from 2023-05-06 13-54-49](https://github.com/ambrose-kibet/Budget-App/assets/66097371/78f568f4-b583-4632-995a-c75a9f4e9b60)
as you can see your category name is blue and not so in the given design, also it has no text decoration